### ISSUE TITLE
Bugs in the Network tool

### ISSUE THREAD

**Issue by CsatiZoltan** [Image-Py/imagepy#82]
For the network tool to work with Python 3.7.6, we need some changes:
- at most `networkx` package version 2.1. Reason: the `node` attribute no longer exists as of version 2.4 (see [here](https://networkx.github.io/documentation/stable/release/release_2.4.html))
- correct variable `n` to `n.astype(np.int16)` in lines 45, 46 of module [graphpen_tol](https://github.com/Image-Py/imagepy/blob/master/imagepy/tools/Network/graphpen_tol.py): https://github.com/Image-Py/imagepy/blob/17aa2927bc1216a8956b6a6f8173817cdd9481bb/imagepy/tools/Network/graphpen_tol.py#L45 https://github.com/Image-Py/imagepy/blob/17aa2927bc1216a8956b6a6f8173817cdd9481bb/imagepy/tools/Network/graphpen_tol.py#L46
  and similarly in module [graphcut_tol](https://github.com/Image-Py/imagepy/blob/17aa2927bc1216a8956b6a6f8173817cdd9481bb/imagepy/tools/Network/graphcut_tol.py): https://github.com/Image-Py/imagepy/blob/17aa2927bc1216a8956b6a6f8173817cdd9481bb/imagepy/tools/Network/graphcut_tol.py#L44 https://github.com/Image-Py/imagepy/blob/17aa2927bc1216a8956b6a6f8173817cdd9481bb/imagepy/tools/Network/graphcut_tol.py#L45

**Comment by CsatiZoltan** [comment]
Remark: the reason why the "Build Graph" tool works in the [standalone Windows](https://github.com/Image-Py/imagepy/releases/download/v0.2/ImagePy-64.rar) version is because it uses `networkx 2.2`. In fact, the previously mentioned issue with the `node` attribute was deprecated since version 2.2, and was removed in version 2.4.

**Comment by yxdragon** [comment]
I just replace all the "node[" by "nodes[", And now every thing is ok for me. I had pushed. have a try?

**Comment by CsatiZoltan** [comment]
Yes, that solves the problem, as expected. You need to make the changes in */menus/Analysis/Skeleton Network/graph_plgs.py* and in */ipyalg/graph/sknw.py*.

The other modification I proposed concerning `linspace` also works.

**Comment by CsatiZoltan** [comment]
Also note that the issue with `.node` also concerns your [sknw](https://github.com/Image-Py/sknw) repository.

**Comment by CsatiZoltan** [comment]
Another error found in [graph_plgs.py](https://github.com/Image-Py/imagepy/blob/master/imagepy/menus/Analysis/Skeleton%20Network/graph_plgs.py): `connected_component_subgraphs` is removed. For a solution, see the related issue https://github.com/rkistner/chinese-postman/issues/21.
